Source/Initialization/InjectorPosition.H

```diff
@@ -134,6 +134,16 @@ struct InjectorPosition
                 z < zmax and z >= zmin);
     }
 
+    // bool: whether the region defined by lo and hi overaps with the plasma region
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    bool
+    overlapsWith (const amrex::XDim3& lo, const amrex::XDim3& hi) const noexcept
+    {
+        return ! (   (xmin > hi.x) || (xmax < lo.x)
+                  || (ymin > hi.y) || (ymax < lo.y)
+                  || (zmin > hi.z) || (zmax < lo.z) );
+    }
+
```

Comment:
Is that required, or do you only need `PlasmaInjector::overlapsWith`?

Reply:
I was following what was done for `insideBounds` here. 

It looks like the difference is that the `PlasmaInjector` is a host-only class, while the `InjectorPosition`, `InjectorMomentum`, etc. are host device. Currently I am only using the `InjectorPosition` version, but for consistency maybe we should have both?

Reply:
It is fine for me, sorry, I just wanted to make sure.